ruleset.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<ruleset name="Centreon Standard">
+  <description>Custom rules defined by Centreon</description>
+
+  <exclude-pattern>centreon-build</exclude-pattern>
+  <exclude-pattern>doc</exclude-pattern>
+  <exclude-pattern>node_modules</exclude-pattern>
+  <exclude-pattern>vendor</exclude-pattern>
+  <exclude-pattern>lib/HTML/QuickForm.php</exclude-pattern>
+  <exclude-pattern>lib/HTML/QuickForm/**/*.php</exclude-pattern>
+  <exclude-pattern>*\.(?!(php$))</exclude-pattern>
+
+  <rule ref="PSR2"/>
+  <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
+    <properties>
+      <property name="ignoreBlankLines" value="false"/>
```

Comment:
According to the [documentation](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizwhitespacesuperfluouswhitespace) _ignoreBlankLines_ should be false by default. Maybe the _properties_ section can be removed ?

Reply:
could I add a rule without properties ?
I'll try ;)